indidriver.c

```diff
@@ -2354,12 +2413,10 @@ void IDSetBLOB(const IBLOBVectorProperty *bvp, const char *fmt, ...)
     printf("  timestamp='%s'\n", timestamp());
     if (fmt)
     {
-        va_list ap;
-        va_start(ap, fmt);
         printf("  message='");
+        // #PS: missing escapeXML_fputs ?
         vprintf(fmt, ap);
         printf("'\n");
-        va_end(ap);
     }
```

Comment:
Sometimes the "entityXML" function is used,
sometimes "escapeXML_fputs", but nothing here.
Right?

With the next review, I can improve it.